wrappers/services.go

```diff
@@ -577,6 +592,7 @@ WantedBy={{.ServicesTarget}}
 		Remain             string
 		KillMode           string
 		KillSignal         string
+		VitalityScore      int
```

Comment:
all the other names use the systemd name?  this should probably be OOMAdjustScore